contrib_bots/lib/DefineBot/DefineBot.py

```diff
@@ -0,0 +1,72 @@
+# See readme.md for instructions on running this code.
```

Comment:
Yes I got that now. Should I move all the files from DefineBot folder to contrib_bots/lib or should I look for another solution?